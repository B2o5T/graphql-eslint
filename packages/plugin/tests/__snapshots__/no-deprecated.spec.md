// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Invalid #1 1`] = `
##### ❌ Error

    > 1 | mutation { something(t: OLD) }
        |                         ^^^ This enum value is marked as deprecated in your GraphQL schema (reason: No longer supported)

##### 💡 Suggestion: Remove \`OLD\` enum value

    1 | mutation { something(t: ) }
`;

exports[`Invalid #2 1`] = `
##### ❌ Error

    > 1 | mutation { something(t: OLD_WITH_REASON) }
        |                         ^^^^^^^^^^^^^^^ This enum value is marked as deprecated in your GraphQL schema (reason: test)

##### 💡 Suggestion: Remove \`OLD_WITH_REASON\` enum value

    1 | mutation { something(t: ) }
`;

exports[`Invalid #3 1`] = `
##### ❌ Error

    > 1 | query { oldField }
        |         ^^^^^^^^ This field is marked as deprecated in your GraphQL schema (reason: No longer supported)

##### 💡 Suggestion: Remove \`oldField\` field

    1 | query {  }
`;

exports[`Invalid #4 1`] = `
##### ❌ Error

    > 1 | query { oldFieldWithReason }
        |         ^^^^^^^^^^^^^^^^^^ This field is marked as deprecated in your GraphQL schema (reason: test)

##### 💡 Suggestion: Remove \`oldFieldWithReason\` field

    1 | query {  }
`;
