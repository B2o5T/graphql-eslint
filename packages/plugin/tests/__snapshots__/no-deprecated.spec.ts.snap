// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
❌ Error 1/1

> 1 | mutation { something(t: OLD) }
    |                         ^^^ This enum value is marked as deprecated in your GraphQL schema (reason: No longer supported)
`;

exports[` 2`] = `
❌ Error 1/1

> 1 | mutation { something(t: OLD_WITH_REASON) }
    |                         ^^^^^^^^^^^^^^^ This enum value is marked as deprecated in your GraphQL schema (reason: test)
`;

exports[` 3`] = `
❌ Error 1/1

> 1 | query { oldField }
    |         ^^^^^^^^ This field is marked as deprecated in your GraphQL schema (reason: No longer supported)
`;

exports[` 4`] = `
❌ Error 1/1

> 1 | query { oldFieldWithReason }
    |         ^^^^^^^^^^^^^^^^^^ This field is marked as deprecated in your GraphQL schema (reason: test)
`;
